griptape/drivers/prompt/openai_chat_prompt_driver.py

```diff
@@ -114,6 +114,7 @@ def _base_params(self, prompt_stack: PromptStack) -> dict:
             "stop": self.tokenizer.stop_sequences,
             "user": self.user,
             "seed": self.seed,
+            "max_tokens": self.max_output_tokens(self.prompt_stack_to_string(prompt_stack)),
```

Comment:
@vasinov @andrewfrench let me know what you think about this. I wonder if we used to get rate limited because we were specifying a `max_tokens` in the order of _input_ tokens instead of output tokens.

This is the only Prompt Driver that had this one-off hack.

Reply:
Oh, wow. I can totally see that being the case.